### PR TITLE
Add information about multiples SIM cards

### LINKED ISSUE #44: Support multiples SIM cards information
The library just support get information from one sim card

From API 22 it's possible get information about multiples simcards:

**More information:**
Android does not support multiple SIMs, at least from the SDK. Device manufacturers who have created multi-SIM devices are doing so on their own. You are welcome to contact your device manufacturer and see if they have an SDK add-on or something that allows you to access the second SIM.

Edit: (15th July, 2015)

Since API 22, you can check for multiple SIMs using SubscriptionManager's method getActiveSubscriptionInfoList().

https://stackoverflow.com/questions/7996197/detect-the-status-of-two-sim-cards-in-a-dual-sim-android-phone

**Official documentation**
https://developer.android.com/reference/android/telephony/SubscriptionManager.html#getActiveSubscriptionInfoList%28%29

### PR DESCRIPTION
**Pull request by Ivans51**
Signed-off-by: Ivan Del Pino <idelpino@teclib.com>

### Changes description

Add information to multiples SIM cards

### Checklist

Please check if your PR fulfills the following specifications:

- [x] Tests for the changes have been added
- [x] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@ivans51 |8|

### References

Closes: https://github.com/flyve-mdm/android-inventory-library/issues/44
Related #N/A
Depends on #N/A

### CODE REVIEW

File: inventory/src/androidTest/java/org/flyve/inventory/SimcardsTest.java [hunk 1]
```diff
@@ -1,42 +1,45 @@
 /**
- *  LICENSE
- * 
- *  This file is part of Flyve MDM Inventory Library for Android.
+ * LICENSE
+ * <p>
```
Comment (ajsb85) [inline inventory/src/androidTest/java/org/flyve/inventory/SimcardsTest.java]: Hi, @Ivans51 
There are extra HTML code in the License header.
And add also your name and email in the headers.

**Review by ajsb85 — CHANGES_REQUESTED** [review]

File: inventory/src/androidTest/java/org/flyve/inventory/SimcardsTest.java (hunk 1 above)
Comment (Ivans51) [reply inventory/src/androidTest/java/org/flyve/inventory/SimcardsTest.java]: Ok

**Comment by ajsb85** [comment]
Hi, @Ivans51 
Could you rebase your feature branch?
BTW, I checked the "Doc" item in the PR body because you was added the documentation in the code 👌

File: inventory/src/main/java/org/flyve/inventory/categories/Simcards.java [hunk 2]
```diff
@@ -87,29 +93,89 @@ public Simcards(Context xCtx) {
...
-                c.put("COUNTRY", new CategoryValue(getCountry(), "COUNTRY", "country"));
-                c.put("OPERATOR_CODE", new CategoryValue(getOperatorCode(), "OPERATOR_CODE", "operatorCode"));
-                c.put("OPERATOR_NAME", new CategoryValue(getOperatorName(), "OPERATOR_NAME", "operatorName"));
-                c.put("SERIAL", new CategoryValue(getSerial(), "SERIAL", "serial"));
-                c.put("STATE", new CategoryValue(getState(), "STATE", "state"));
-                c.put("LINE_NUMBER", new CategoryValue(getLineNumber(), "LINE_NUMBER", "lineNumber"));
-                c.put("SUBSCRIBER_ID", new CategoryValue(getSubscriberId(), "SUBSCRIBER_ID", "subscriberId"));
-
-                this.add(c);
+            if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP_MR1) {
```
Comment (rafaelje) [inline inventory/src/main/java/org/flyve/inventory/categories/Simcards.java:99]: 👍

**Review by rafaelje — APPROVED** [review]

**Review by ajsb85 — APPROVED** [review]

**Comment by ajsb85** [comment]
Hi, @Ivans51 
The `refactor `commit message type is just for code, not for documentation.

**Pull request merged**